### PR TITLE
feat(1271): templates & commands list page

### PR DESCRIPTION
**Pull request by DekusDenial**
## Context
Utilize faster query option to improve templates' and commands' list page

## Objective

- Implement newer design for the list page while keeping the diff minimal and remove several soon-to-be unused components
- Fix the namespace issue with templates that have changed their namespaces

image

## Related links
Related to https://github.com/screwdriver-cd/screwdriver/issues/1271

### CODE REVIEW

File: app/components/tc-collection-list/styles.scss [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+      text-overflow: ellipsis;
+      overflow: auto;
+    }
+
+    td {
+      padding: 15px;
+    }
+
+    tbody tr {
+      background: #fff;
```
Comment (tkyi) [inline app/components/tc-collection-list/styles.scss]: $sd-white

File: app/components/tc-collection-list/styles.scss [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  .control-icon {
+    position: absolute;
+    top: 50%;
+    right: 30px;
+    @include transform(translateY(-50%));
+  }
+
+  .collection-list {
+    th {
+      color: #9A9A9A;
```
Comment (tkyi) [inline app/components/tc-collection-list/styles.scss]: $grey-600

File: app/components/tc-collection-list/template.hbs [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    <a href="http://docs.screwdriver.cd/user-guide/{{routes.prefix}}" title="More Information" target="_blank" class="link pull-right">{{collectionType}} Docs {{fa-icon "question-circle"}}</a>
+  </h4>
+  <div class="collection-description">{{yield}}</div>
+</header>
+
+<section class="row">
+  <h5>
+    <b>Screwdriver {{collectionType}}</b>
+    <span class="pull-right">
+      <span class="text-uppercase" style="color: #9A9A9A;">Total </span><span>{{filteredModel.length}}</span>
```
Comment (tkyi) [inline app/components/tc-collection-list/template.hbs]: do you still need this here: `style="color: #9A9A9A;`

File: tests/helpers/responsive.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
+/* eslint-disable */
```
Comment (tkyi) [inline tests/helpers/responsive.js]: probably want to remove this

File: tests/integration/components/tc-collection-list/component-test.js [hunk 5]
```diff
@@ -0,0 +1,55 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+// import { setBreakpoint } from 'ember-responsive/test-support';
```
Comment (tkyi) [inline tests/integration/components/tc-collection-list/component-test.js]: commented line

**Comment by DekusDenial** [comment]
added blurred gif

**Pull request merged**